simulator/docs/environment-variables.md

```diff
@@ -28,6 +32,9 @@ origin for `CORS_ALLOWED_ORIGIN_LIST`.
 resources". This variable is used to find Kubeconfig required to
 access your cluster for importing resources to scheduler simulator.
 
+`KUBE_APISEVER_URL`: This is the URL of kube-apiserver which the
+simulator starts internally. Its default value is `http://{simulator-cluster-ip}:8080`.
```

Comment:
> Its default value is `http://{simulator-cluster-ip}:8080`.

I don't see where we set this. Anyways, I guess we should make it required configuration, shouldn't we. Because we no longer start kube-apiserver internally and the simulator cannot run without kube-apiserver outside

Reply:
OK, I agree with your opinion. I''ll make this value required value.